### PR TITLE
Add TOC as sidebar to UG, DG

### PR DESCRIPTION
**Pull request by zichen-3974**
View page [here](https://zichen-3974.github.io/tp/)

- TOC appears on right side of user guide and developer guide. 
- All drop-downs are not expanded by default

### CODE REVIEW

File: docs/_includes/tocUg.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <li><a href="#deleting-a-contact-delete" id="markdown-toc-deleting-a-contact-delete">Deleting a contact: <code class="language-plaintext highlighter-rouge">delete</code></a></li>
+        <li><a href="#tagging-a-module-to-contact--tag" id="markdown-toc-tagging-a-module-to-contact--tag">Tagging a module to contact : <code class="language-plaintext highlighter-rouge">tag</code></a></li>
+        <li><a href="#untagging-a-module-from-an-existing-contact--untag" id="markdown-toc-untagging-a-module-from-an-existing-contact--untag">Untagging a module from an existing contact : <code class="language-plaintext highlighter-rouge">untag</code></a></li>
+        <li><a href="#locating-persons-by-keyowrds--find" id="markdown-toc-locating-persons-by-keyowrds--find">Locating persons by keyowrds : <code class="language-plaintext highlighter-rouge">find</code></a></li>
+        <li><a href="#view-contacts--view" id="markdown-toc-view-contacts--view">View Contacts : <code class="language-plaintext highlighter-rouge">view</code></a></li>
+    </ul>
+</details>
+<a href="#faq" id="markdown-toc-faq">FAQ</a>
```
Comment (russelltankaimin) [inline docs/_includes/tocUg.html:18]: Is it intended for FAQ and Command Summary to appear side-by-side?

Comment (russelltankaimin) [reply docs/_includes/tocUg.html:18]: I am not sure if I id-ed the correct place here.

**Review by mynameizzhafeez — COMMENTED** [review]
LGTM. When making UG/DG changes, could you deploy it on your fork? So that we can see it on our end :)

File: docs/_includes/tocUg.html (hunk 1 above)
Comment (russelltankaimin) [reply docs/_includes/tocUg.html]: LGTM now, thanks for making the UG look nice!👍👍

File: docs/_layouts/default.html
[the remark this answers is not in the document]
Comment (mynameizzhafeez) [reply docs/_layouts/default.html:1]: Alright. I understsand :)

**Review by mynameizzhafeez — APPROVED** [review]
LGTM.

**Pull request merged**